.github/workflows/ci.yaml

```diff
@@ -75,18 +71,15 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Get prefsrc
-        run: |
-          echo "IPADDR=$(ip -4 -j route get 2.2.2.2 | jq -r '.[] | .prefsrc')" >> $GITHUB_ENV
       - name: Setup operator environment
         # TODO: Replace with custom image on self-hosted runner
         uses: charmed-kubernetes/actions-operator@main
         with:
-          juju-channel: 3.5/stable
+          juju-channel: 3.6/stable
           provider: microk8s
-          channel: 1.28-strict/stable
+          channel: 1.32-strict/stable
           microk8s-group: snap_microk8s
-          microk8s-addons: "hostpath-storage dns minio metallb:${{ env.IPADDR }}-${{ env.IPADDR }}"
+          microk8s-addons: "hostpath-storage dns minio"
```

Comment:
question: why do we remove this?

Reply:
I wanted to try running the CI without it, see if it improves the reliability of our test setup. Since the hub does not need it, I figured it would be ok to have the deployment as light as possible